app/css/interceptor.css

```
@@ -343,6 +343,13 @@ a.dropdown-item.is-active, button.dropdown-item.is-active {
 	padding: 0;
 	padding-left: 0.75rem;
 	padding-right: 0.75rem;
+	background-color: transparent;
+}
+
+.card-header-icon[disabled] {
```

you might be targeting "disabled state" here and not presence of "disabled" attribute on the button

```suggestion
.card-header-icon:disabled {
```